<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormProjectX</class>
 <widget class="QMainWindow" name="FormProjectX">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>800</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>ProjectX</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>10</y>
      <width>262</width>
      <height>401</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="title">
     <string>Дерево проекта</string>
    </property>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>221</y>
       <width>262</width>
       <height>161</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Список связей</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>29</y>
       <width>262</width>
       <height>122</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string/>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>264</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <italic>false</italic>
      </font>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>155</y>
       <width>65</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Действия</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>420</y>
      <width>261</width>
      <height>212</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="title">
     <string>Свойства</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="View">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>781</width>
      <height>631</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>352</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QToolButton" name="inP">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>33</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string comment="Материальная точка" extracomment="Материальная точка">Мат.точка</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>Point.png</normaloff>Point.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="inR">
     <property name="geometry">
      <rect>
       <x>49</x>
       <y>9</y>
       <width>33</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>Line.png</normaloff>Line.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="inW">
     <property name="geometry">
      <rect>
       <x>89</x>
       <y>9</y>
       <width>33</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>Wheel.png</normaloff>Wheel.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>10</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Детали</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>411</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <zorder>groupBox_7</zorder>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>381</x>
      <y>10</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Движение</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Anim">
    <property name="geometry">
     <rect>
      <x>571</x>
      <y>720</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Анимация</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopAnim">
    <property name="geometry">
     <rect>
      <x>717</x>
      <y>720</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Стоп</string>
    </property>
   </widget>
   <widget class="QPushButton" name="returnAnim">
    <property name="geometry">
     <rect>
      <x>644</x>
      <y>720</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Повторить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delAnim">
    <property name="geometry">
     <rect>
      <x>498</x>
      <y>720</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_8">
    <property name="geometry">
     <rect>
      <x>819</x>
      <y>644</y>
      <width>262</width>
      <height>98</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="title">
     <string>Текущие координаты</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="ExitMenu"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="timeSetting"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Помощь</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="ExitMenu">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="timeSetting">
   <property name="text">
    <string>Время анимации</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
